src/app/api/chores/route.js

```diff
@@ -7,6 +7,115 @@ import { checkMembership } from "../_utils";
 
 export const dynamic = "force-dynamic";
 
+/**
```

Comment:
This comment is what we add right? And then swagger takes it and displays it nicely at the endpoint with all the documentation?

Reply:
Correct!